src/containers/Amendments/test/amendmentsTable.test.js

```diff
@@ -0,0 +1,29 @@
+import { mount } from 'enzyme'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { I18nextProvider } from 'react-i18next'
+import i18n from '../../../i18n/testConfig'
+import { AmendmentsTable } from '../AmendmentsTable'
+import amendments from './mockAmendmentsProcessed.json'
+
+/* eslint-disable react/jsx-props-no-spreading */
+const createWrapper = (props = {}) =>
+  mount(
+    <Router>
+      <I18nextProvider i18n={i18n}>
+        <AmendmentsTable {...props} />
+      </I18nextProvider>
+    </Router>,
+  )
+
+describe('Amendments table', () => {
+  it('renders without crashing', () => {
+    const wrapper = createWrapper()
+    wrapper.unmount()
+  })
+
+  it('renders all parts', () => {
+    const wrapper = createWrapper({ amendments })
+    expect(wrapper.find('tr').length).toBe(amendments.length + 1)
```

Comment:
I would like to see some more detailed tests that check specific column's data.

Reply:
The detailed test for each column (for both enabled and voting amendments) is in the `amendments.test.js` file. I'm just following the same structure as the tests in the `Network` folder here.